model_compression_toolkit/core/common/graph/base_graph.py

```diff
@@ -12,10 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-from collections import namedtuple
-
+# from collections import namedtuple
```

Comment:
Please remove commented-out import

Reply:
done, thx

Reply:
I see that the comment still exists, can you remove it please?